ai-service/src/main/webapp/WEB-INF/ibm-web-ext.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>		
```

Comment:
this file can be removed, the only effect it's having now is setting the app context root to `/`, but I don't think anything else depends on the context root. If it does, we can set the context root in server config to be `/`